### PR TITLE
Create and push docker image on new tag

### PR DESCRIPTION
**Pull request by knikolla**
Uses GitHub actions to build and push a new docker image to
Docker Hub when a new tag is pushed.

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
```
Comment (larsks) [inline .github/workflows/release.yaml]: You don't need to set up qemu unless you're planning on building multi-architecture images.

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      - name: Get the version
+        id: vars
+        run: echo ::set-output name=tag::${GITHUB_REF#refs/tags/}
+
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: massopencloud/horizon-onboarding:${{ steps.vars.outputs.tag }}
```
Comment (larsks) [inline .github/workflows/release.yaml]: If you make the image name a repository secret, you can re-use this workflow verbatim in other repositories.

Comment (knikolla) [reply .github/workflows/release.yaml]: Good idea. Will do.

File: .github/workflows/release.yaml (hunk 1 above)
Comment (knikolla) [reply .github/workflows/release.yaml]: I decided on leaving this here initially, and seeing how much time it adds. Depending on what we do with the Power9s, we may or may not want multiarch builds yet.

**Review by larsks — APPROVED** [review]

**Pull request merged**